### PR TITLE
Unix time converter update

### PR DESCRIPTION
**Pull request by lucolvin**
Fixed formatting and added an if statement to handle 0 seconds ago because it currently say ago at 0 seconds.

### CODE REVIEW

**Comment by YourAverageTechBro** [comment]
@lucolvin could you fix the error on the deployment?

**Comment by lucolvin** [comment]
@YourAverageTechBro I am unsure of what is causing the error everything should work. There is something happening in vercel that I cannot see.

**Comment by YourAverageTechBro** [comment]
I think you are missing a closing bracket in the time converter. Please check again.

File: src/app/tools/unix-time-converter/UnixTimeConverterComponent.tsx [hunk 1]
```diff
@@ -181,6 +181,8 @@ export default function UnixTimeConverterComponent({
         if (Number.isNaN(timeInMilliseconds)) {
           throw new Error("Time string is not a number");
         }
+        if (timeInMilliseconds === Date.now()) {
+          setRelativeTime("0 seconds ago");
```
Comment (timagixe) [inline src/app/tools/unix-time-converter/UnixTimeConverterComponent.tsx:185]:
```suggestion
        if (timeInMilliseconds === Date.now()) {
          setRelativeTime("0 seconds ago");
        }
```

**Pull request merged**